src/puppetlabs/trapperkeeper/authorization/rules.clj

```diff
@@ -15,6 +15,7 @@
    :type (schema/enum :string :regex)
    :path Pattern
    :method (schema/enum :get :post :put :delete :head :any)
+   (schema/optional-key :allow_unauthenticated) schema/Bool
```

Comment:
What do you think about adding an "invalid config" test for this new field to ensure that it's of the proper type?

See: https://github.com/puppetlabs/trapperkeeper-authorization/blob/master/test/puppetlabs/trapperkeeper/services/authorization/authorization_core_test.clj#L47

Or do we have near-term plans to change how validation is done? Can't remember, but I ended up writing these tests for the query-param stuff.